src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
```

Comment:
The `.filter()` method is used appropriately to remove any strings that do not contain a colon, which would not be valid CSS declarations.